packages/webauthn-pcd/README.md

```diff
@@ -0,0 +1,3 @@
+# `@pcd/webauthn-pcd`
+
+A wrapper around WebAuthn authentication verification as specified by the [W3C protocol](https://www.w3.org/TR/webauthn-2/#sctn-verifying-assertion).
```

Comment:
i think it would be neat to get a few paragraphs describing what webauthn is, and some potential interesting use-cases (like proof of faceid)

Reply:
added more info and some use cases!